README.md

```diff
@@ -80,29 +80,6 @@ binaries.
 // +heroku install example.com/example-server example.com/example-worker
 ```
 
-## Development
-
-### Dependencies
-
-This buildpack relies on [heroku/libcnb.rs](libcnb) to compile buildpacks. All
-[libcnb.rs dependencies](https://https://github.com/heroku/libcnb.rs#development-environment-setup) will need to be setup prior to building or testing this buildpack.
-
-### Building
-
-1. Run `cargo check` to download dependencies and ensure there are no
-   compilation issues.
-1. Build the buildpack with `cargo libcnb package`.
-1. Use the buildpack to build an app: `pack build go-example --buildpack packaged/x86_64-unknown-linux-musl/debug/heroku_go --path /path/to/go-app`
-
-### Testing
-
-- `cargo test` performs Rust unit tests.
-- `cargo test -- --ignored` performs all integration tests.
-
-## Releasing
-
-[Deploy Cloud Native Buildpacks](https://github.com/heroku/languages-team/blob/main/languages/cnb/deploy.md)
-
 ## License
 
 Licensed under the BSD 3-clause License. See [LICENSE](./LICENSE) file.
```

Comment:
```suggestion
Licensed under the BSD 3-clause License. See [LICENSE](./LICENSE.txt) file.
```

Reply:
Alternatively, rename the file to `LICENSE` like most of the other CNB buildpacks